frontend/src/components/nodes/ValidatorTelemetryRow.tsx

```diff
@@ -56,7 +56,11 @@ const ValidatorTelemetryRow: FC<Props> = ({
             <Col className="validator-nodes-text uptime">
               {producedBlocks !== undefined && expectedBlocks !== undefined ? (
                 <>
-                  {((producedBlocks / expectedBlocks) * 100).toFixed(3)}% &nbsp;
+                  {expectedBlocks === 0
+                    ? "n/a"
```

Comment:
Let's display N/A as we already do in several places here.

```suggestion
                    ? t("common.state.not_available")
```

Or can just add a condition to 57 line `|| expectedBlocks === 0`.

@frol @luixo WDYT?

Reply:
Totally makes sense
I'm not used to i18n yet